pkgs/development/libraries/science/math/mongoose/default.nix

```
@@ -16,17 +19,27 @@ stdenv.mkDerivation rec {
     sha256 = "0ymwd4n8p8s0ndh1vcbmjcsm0x2cc2b7v3baww5y6as12873bcrh";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ScottKolo/Mongoose/commit/39f4a0059ff7bad5bffa84369f31839214ac7877.patch";
+      sha256 = "sha256-V8lCq22ixCCzLmKtW6bUL8cvJFZzdgYoA4BFs4xYd3c=";
+    })
+  ];
+
   nativeBuildInputs = [
     cmake
   ];
 
+  cmakeFlags = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
```

There is only one option, i.e. `lib.option` is sufficient:
```suggestion
  cmakeFlags = lib.optional (stdenv.isDarwin && stdenv.isAarch64)
```

---

Done. Thanks!